src/algebra/lie/basic.lean

```diff
@@ -120,6 +125,12 @@ begin
   unfold has_scalar.smul, rw gsmul_neg,
 end
 
+lemma jacobi (x y z : L) : ⁅x, ⁅y, z⁆⁆ + ⁅y, ⁅z, x⁆⁆ + ⁅z, ⁅x, y⁆⁆ = 0 :=
+begin
+  have h : ⁅z,⁅y,x⁆⁆ = -⁅z,⁅x,y⁆⁆, { rw ← lie_neg, congr, rw lie_skew, },
+  rw [←lie_skew, lie_ring.lie_lie, h], abel,
```

Comment:
Do you need the `lie_ring.` prefix? Can't we have a global `lie_lie`?

Reply:
Done!